server/src/main/java/org/opensearch/action/resync/TransportResyncReplicationAction.java

```diff
@@ -109,7 +110,8 @@ public TransportResyncReplicationAction(
             EXECUTOR_NAME_FUNCTION,
             true, /* we should never reject resync because of thread pool capacity on primary */
             indexingPressureService,
-            systemIndices
+            systemIndices,
+            NoopTracer.INSTANCE
```

Comment:
@rayshrey  I believe these actions are all covered by existing instrumentation, right? we should not pass NOOP tracer than. the whole purpose of instrumenting the base write action is to cover a whole family of actions, including the indexing path.